src/main/java/org/zephyrsoft/sdb2/gui/MainWindow.java

```diff
@@ -1157,8 +1259,13 @@ protected void handleJumpToPresentedSong() {
 	 * @param song
 	 */
 	public void present(Song song) {
-		presentListSelected = song;
-		handleSongPresent();
+		Song currentlyPresentedSong = controller.getCurrentlyPresentedSong();
+		if (currentlyPresentedSong != null && !currentlyPresentedSong.equals(song)
+			|| currentlyPresentedSong == null && song != null) {
+			presentListSelected = song;
+			handleSongPresent();
+			handleJumpToPresentedSong();
```

Comment:
This should not be done automatically, the list of all available songs is not connected to the presentation list. If the selection changes on every change of the presented song, the user could get the impression that it _is_ connected somehow.

Reply:
Oh yes, thought this would only select the current song in the playlist, but it selects the song in the songlist too. So I removed it now.